modules/internal/DefaultSparse.chpl

```diff
@@ -275,7 +281,7 @@ module DefaultSparse {
         _bulkGrow();
 
         var indIdx = _indices.domain.low;
-        var prevIdx = parentDom.low - 1;
+        var prevIdx = parentDom.low;
```

Comment:
I was thinking that `-1` makes sure that `prevIdx` is initially an index that can never be contained in this sparse domain. And I am probably right that it was the intention when this is created.

But then, I looked at the code and saw that `prevIdx` is unused in this function. Probably the intern implemented this (*cough* me) forgot to remove it :) You might as well just remove it here, or leave it untouched and I can clean my own mess.

Reply:
Actually I think it needs to be in here.  Otherwise that continue on 292 could happen.  I am putting the -1 back in there.  Good catch!